assets/src/defaultPalette.js

```diff
@@ -22,11 +22,14 @@ const defaultPalette = {
   link: {
     main: '#0000EE'
   },
-  /*  The 'info' color is used to confirm a submitted or stored by system state .
-    For now, this should be used directly through theme.palette.info.main instead of the color prop
-    (i.e. color='info'), which isn't fully supported. */
+  /*  The 'info' color is used to confirm a submitted or stored by system state. */
```

Comment:
@zqian, because `info` is a standard Material UI color name, I believe you can use it with the `color` prop in Material UI components.